vss_invoice_mgmt/src/constants/productmeasureunit.js

```diff
@@ -1 +1 @@
-export const productmeasures = ["Ltr", "Grams", "KG", "Piece"];
```

Comment:
Why you pushed the product master related constansts?

Reply:
removed